setup.py

```diff
@@ -7,17 +7,12 @@
 import os
 from setuptools import setup
 
-cwd = os.path.dirname(os.path.abspath(__name__))
-
-long_description = open(os.path.join(cwd, 'README.rst'), 'r').read()
```

Comment:
I was wondering, is the problem also fixed if you open   the file using `utf-8` encoding? like

``` 
 long_description = open(os.path.join(cwd, 'README.rst'), 'r', encoding='utf-8').read() 
```

Reply:
I was not sure you are willing to keep compatibility with python2, so I suggested the way using `setup.cfg` that will also work with python2. If you are not willing, of course the way using `encoding='utf-8'` is also possible.

Reply:
I like that you thought about backward compatibility. But we just decided to drop Python 2 support, see #86 . So we can use the `encoding` argument of `open()`.